cli/command/telemetry_docker.go

```diff
@@ -180,7 +180,7 @@ func toWslPath(s string) string {
 	if !ok {
 		return ""
 	}
-	return fmt.Sprintf("mnt/%s%s", drive, p)
+	return fmt.Sprintf("mnt/%s%s", strings.ToLower(drive), p)
```

Comment:
should we put this in the `parseUNCPath` function?

Reply:
Ah, never mind; probably bad suggestion, as this conversion is indeed more related to wsl2 than UNC paths